default/methods/task/task/task_user_add.py

```diff
@@ -17,19 +18,26 @@ def api_task_user_add(project_string_id, task_id):
             }}
         ]
 
+        result = {}
+
         log, input, untrusted_input = regular_input.master(request = request,
                                                            spec_list = spec_list)
         if len(log["error"].keys()) >= 1:
             return jsonify(log = log), 400
+        
+        for user in input['user_id']:
```

Comment:
for the `input['user_id']` I would just rename it to `input['user_id_list']` just to make it more explicit for other develiper that this key is a list.

Reply:
Yeah, makes sense